templates/packageVersion.mustache

```diff
@@ -66,6 +60,31 @@
           <p>None</p>
           {{/cabalFileRevisions}}
         </dd>
+        {{#pkgDesc.library}}
+        <dt class="col-sm-3">Dependencies</dt>
```

Comment:
This is a bit weird: you say "dependencies" here unqualified, and then for the other components you just say their name and don't say "dependencies". I would be inclined to either make this a more structured section under a "dependencies" header or say: "library foo dependencies", "test-suite bar dependencies", "executable baz" dependencies".